### PR TITLE
Try upgrading packages from the base image

### PR DESCRIPTION
**Pull request by davisagli**
See https://pythonspeed.com/articles/security-updates-in-docker/ for why to do this.

### CODE REVIEW

**Comment by Rudd-O** [comment]
Thank you, this should absolutely be done.

File: Dockerfile [hunk 1]
```diff
@@ -10,6 +10,7 @@ ENV EXTRA_PACKAGES="relstorage==3.4.5 psycopg2==2.9.3 python-ldap==3.4.0"
 
 RUN <<EOT
     apt-get update
+    apt-get -y upgrade
```
Comment (jensens) [inline Dockerfile:13]: I don't think we need it in the intermediate image.

**Review by jensens — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (davisagli) [reply Dockerfile:13]: @jensens If there is a security fix for a build tool, I would like it to be applied before I run the build tool and the exploit steals my build secrets.

Comment (jensens) [reply Dockerfile:13]: You do a fresh install of the build tools anyway. Slim does not contain much. OTOH it does not hurt much ;)

**Review by jensens — APPROVED** [review]

**Comment by jensens** [comment]
Is this WIP by intend?

**Comment by davisagli** [comment]
It was WIP until I could see the result of the tests. Marking ready for review now.

**Pull request merged**